packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -312,7 +312,7 @@ abstract class Route<T> {
   ///
   /// This is called by [didPop] and in response to
   /// [NavigatorState.pushReplacement]. If [didPop] was not called, then the
-  /// [Navigator.finalizeRoute] method must be called immediately, and no exit
+  /// [NavigatorState.finalizeRoute] method must be called immediately, and no exit
```

Comment:
It's probably worth adding docs to the class docs of `NavigatorState` that says you can get a reference to the class by calling `Navigator.of`